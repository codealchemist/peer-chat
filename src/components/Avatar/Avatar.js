import React from 'react'
import Icon from '@material-ui/core/Icon'

const Avatar = ({ avatar }) => (
  <Icon style={{ fontSize: '50px' }}>{avatar}</Icon>
)

export default Avatar
