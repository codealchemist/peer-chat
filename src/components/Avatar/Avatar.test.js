import React from 'react'
import { mount } from 'enzyme'
import Avatar from './Avatar'

describe('Avatar', () => {
  it('should render', () => {
    const wrapper = mount(<Avatar avatar="rock" />)
    expect(wrapper).toMatchSnapshot()
  })
})
