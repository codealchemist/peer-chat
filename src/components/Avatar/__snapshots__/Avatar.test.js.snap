// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should render 1`] = `
<Avatar
  avatar="rock"
>
  <WithStyles(Icon)
    style={
      Object {
        "fontSize": "50px",
      }
    }
  >
    <Icon
      classes={
        Object {
          "colorAction": "MuiIcon-colorAction-4",
          "colorDisabled": "MuiIcon-colorDisabled-6",
          "colorError": "MuiIcon-colorError-5",
          "colorPrimary": "MuiIcon-colorPrimary-2",
          "colorSecondary": "MuiIcon-colorSecondary-3",
          "fontSizeInherit": "MuiIcon-fontSizeInherit-7",
          "fontSizeLarge": "MuiIcon-fontSizeLarge-9",
          "fontSizeSmall": "MuiIcon-fontSizeSmall-8",
          "root": "MuiIcon-root-1",
        }
      }
      color="inherit"
      component="span"
      fontSize="default"
      style={
        Object {
          "fontSize": "50px",
        }
      }
    >
      <span
        aria-hidden="true"
        className="material-icons MuiIcon-root-1"
        style={
          Object {
            "fontSize": "50px",
          }
        }
      >
        rock
      </span>
    </Icon>
  </WithStyles(Icon)>
</Avatar>
`;
